virt/virt-2-5-release-notes.adoc

```diff
@@ -232,3 +232,6 @@ To unlock the account, log in to the Manager and enter the following command:
 ----
 $ ovirt-aaa-jdbc-tool user unlock admin
 ----
+
+//New known issue for 2.5.8
+* If you are logged in to the {product-title} as a user with `basic-user` privileges, trying to retrieve guest agent information by running either `virtctl guestosinfo <vmi_name>` or `oc describe vmi` fails. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2000464[*BZ#2000464*])
```

Comment:
I am not sure if it is accurate that `oc describe vmi` is not working. I know that `virtctl guestosinfo` was definitely affected and that is what I fixed, but I did not see any issues with `describe`.

Reply:
@kbidarkar Can you please respond to Roman's comment?

Reply:
If `oc describe vmi` works, would it be accurate to say in the release note - "As a workaround, use the `oc describe vmi` command to fetch guest agent data."?

Reply:
> If `oc describe vmi` works, would it be accurate to say in the release note - "As a workaround, use the `oc describe vmi` command to fetch guest agent data."?

For getting limited data, yes. It would only provide a subset (Operating system info). So things like file system info, timezone, users, ... would not be there.

Reply:
@sjhala-ccs , I just re-checked the test run logs and @rmohr  is right, this affects only `virtctl guestosinfo` , tests related to `oc describe vmi` that is `test_vmi_guest_agent_info_*` all have PASSED. 
So Let's drop `oc describe vmi`. Thanks @rmohr for the catch.